lighthouse-core/gather/gather-runner.js

```diff
@@ -453,12 +453,18 @@ class GatherRunner {
    */
   static async initializeBaseArtifacts(options) {
     const hostUserAgent = (await options.driver.getBrowserVersion()).userAgent;
-
+    const pageUserAgent = await options.driver.getPageUserAgent();
     const {emulatedFormFactor} = options.settings;
+
     // Whether Lighthouse was run on a mobile device (i.e. not on a desktop machine).
     const IsMobileHost = hostUserAgent.includes('Android') || hostUserAgent.includes('Mobile');
+    // IsMobilePage indicates whether the page is mobile (with any pre-applied mobile emulation or not)
+    // In DevTools, emulation is applied before Lighthouse starts (to deal with viewport emulation bugs)
+    // emulatedFormFactor will be 'none', but it's certainly still TestedAsMobileDevice
+    const IsMobilePage = pageUserAgent.includes('Android') || pageUserAgent.includes('Mobile');
     const TestedAsMobileDevice = emulatedFormFactor === 'mobile' ||
-      (emulatedFormFactor !== 'desktop' && IsMobileHost);
+      (emulatedFormFactor !== 'desktop' && IsMobileHost) ||
```

Comment:
should we align both of these on `emulatedFormFactor === 'none'`? AFAIK the `!== desktop` and `=== none` should be functionally equivalent

Reply:
actually now that I think about it if we're on a mobile host but the page UA still isn't mobile we probably shouldn't say `TestedAsMobileDevice`?

maybe we just want `emulatedFormFactor === 'none' && isMobilePage`